ground-mo-proxy/src/main/java/esa/mo/nmf/groundmoproxy/helper/ArchiveHelper.java

```diff
@@ -0,0 +1,100 @@
+/* ----------------------------------------------------------------------------
+ * Copyright (C) 2015      European Space Agency
+ *                         European Space Operations Centre
+ *                         Darmstadt
+ *                         Germany
+ * ----------------------------------------------------------------------------
+ * System                : ESA NanoSat MO Framework
+ * ----------------------------------------------------------------------------
+ * Licensed under European Space Agency Public License (ESA-PL) Weak Copyleft – v2.4
+ * You may not use this file except in compliance with the License.
+ *
+ * Except as expressly set forth in this License, the Software is provided to
+ * You on an "as is" basis and without warranties of any kind, including without
+ * limitation merchantability, fitness for a particular purpose, absence of
+ * defects or errors, accuracy or non-infringement of intellectual property rights.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ----------------------------------------------------------------------------
+ */
+package esa.mo.nmf.groundmoproxy.helper;
+
+import esa.mo.nmf.groundmoproxy.entities.LastArchiveSync;
+import java.util.concurrent.Semaphore;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.persistence.EntityManager;
+
+/**
+ * Archive helper class.
+ *
+ * @author Lukasz.Stochlak
+ */
+public class ArchiveHelper
```

Comment:
Can you give a better name to this class?
I believe there is already a class called either ArchiveHelper or HelperArchive.
To me this does not seem like an helper but a new mechanism to find the last sync.
Maybe you should call it: LastSyncHelper or something similar?

Reply:
Done